File: app/src/main/java/com/kacper/and_krakgo/InputTypes.java
package com.kacper.and_krakgo;

/**
 * Created by kacper on 22/10/2017.
 */

public enum InputTypes {
    PASSWORD(6,30),
    EMAIL(4,70),
    NAME(1, 70),
    SURNAME(1,70),
    RETYPEPASSWORD(6,30);

    private int mMinimumLength;
    private int mMaximumLength;

    InputTypes(int mMinimumLength, int mMaximumLength) {
        this.mMinimumLength = mMinimumLength;
        this.mMaximumLength = mMaximumLength;
    }

    public int getmMinimumLength() {
        return mMinimumLength;
    }

    public int getmMaximumLength() {
        return mMaximumLength;
    }
}

File: app/src/main/java/com/kacper/krakgo/model/enums/InputTypes.kt
package com.kacper.krakgo.model.enums

/**
 * Created by kacper on 22/10/2017.
 */

enum class InputTypes(private val mMinimumLength: Int, private val mMaximumLength: Int) {
    PASSWORD(6, 30),
    EMAIL(4, 70),
    NAME(1, 70),
    SURNAME(1, 70),
    RETYPEPASSWORD(6, 30);

    fun getMinimumLength(): Int {
        return mMinimumLength
    }

    fun getMaximumLength(): Int {
        return mMaximumLength
    }
}
